sdk/keyvault/azure-security-keyvault-administration/README.md

```diff
@@ -0,0 +1,127 @@
+# Azure Key Vault Administration library for Java
+
+## Getting started
+### Adding the package to your project
+Maven dependency for the Azure Key Vault Administration client library. Add it to your project's POM file.
```

Comment:
There are multiple clients, so probably just drop "client" and say "... Administration library."